clients/include/testing_tbmv_strided_batched.hpp

```diff
@@ -0,0 +1,270 @@
+            handle, uplo, transA, diag, M, K, dA, lda, stride_A, dx, incx, stride_x, batch_count));
+
+        // copy output from device to CPU
+        CHECK_HIP_ERROR(hipMemcpy(hx_1, dx, sizeof(T) * size_x, hipMemcpyDeviceToHost));
+
+        // CPU BLAS
+        cpu_time_used = get_time_us();
+        for(int b = 0; b < batch_count; b++)
+            cblas_tbmv<T>(
+                uplo, transA, diag, M, K, hA + b * stride_A, lda, hx_gold + b * stride_x, incx);
+
+        cpu_time_used = get_time_us() - cpu_time_used;
+        cblas_gflops  = batch_count * tbmv_gflop_count<T>(M, K) / cpu_time_used * 1e6;
+
+        if(arg.unit_check)
+        {
+            unit_check_general<T>(1, M, batch_count, abs_incx, stride_x, hx_gold, hx_1);
+        }
+
+        if(arg.norm_check)
+        {
+            rocblas_error_1
+                = norm_check_general<T>('F', 1, M, abs_incx, stride_x, batch_count, hx_gold, hx_1);
+        }
+    }
+
+    if(arg.timing)
+    {
+        int number_cold_calls = 2;
+        int number_hot_calls  = arg.iters;
+        CHECK_ROCBLAS_ERROR(rocblas_set_pointer_mode(handle, rocblas_pointer_mode_host));
+
+        for(int iter = 0; iter < number_cold_calls; iter++)
+        {
+            rocblas_tbmv_strided_batched<T>(handle,
+                                            uplo,
+                                            transA,
+                                            diag,
+                                            M,
+                                            K,
+                                            dA,
+                                            lda,
+                                            stride_A,
+                                            dx,
+                                            incx,
+                                            stride_x,
+                                            batch_count);
+        }
+
+        gpu_time_used = get_time_us(); // in microseconds
+
+        for(int iter = 0; iter < number_hot_calls; iter++)
+        {
+            rocblas_tbmv_strided_batched<T>(handle,
+                                            uplo,
+                                            transA,
+                                            diag,
+                                            M,
+                                            K,
+                                            dA,
+                                            stride_A,
+                                            lda,
+                                            dx,
+                                            incx,
+                                            stride_x,
+                                            batch_count);
+        }
+
+        gpu_time_used     = (get_time_us() - gpu_time_used) / number_hot_calls;
+        rocblas_gflops    = batch_count * tbmv_gflop_count<T>(M, K) / gpu_time_used * 1e6;
+        rocblas_bandwidth = batch_count * (1.0 * M * M) * sizeof(T) / gpu_time_used / 1e3;
```

Comment:
:) 1.0

Reply:
This calc was wrong to begin with, should be fixed now if my math is right.

Reply:
Updated after discussion with @YvanMokwinski and @mahmoodw.

Bandwidth = loads from A + loads from x
loads from x = m
loads from A = sigma(m-i) from i = 0 : k1
loads from A = m * k1 - ((k1 * (k1 + 1)) / 2) + m

therefore bandwidth = (m * k1 - ((k1 * (k1 + 1)) / 2) + 2*m
Where k1 is min(k, m) since when k > m we aren't using the extra values that exceed the size of the matrix.

Reply:
Edit: I forgot the store to x, so add another m to get: therefore bandwidth = ( (m * k1 - ((k1 * (k1 + 1)) / 2) + 3*m ) * sizeof(T) / time